AdminUi/apps/admin_ui/lib/core/modals/add_quota_dialog.dart

```diff
@@ -103,7 +103,7 @@ class _AddQuotaDialogState extends State<_AddQuotaDialog> {
                 enabled: !_saving,
                 decoration: InputDecoration(
                   border: const OutlineInputBorder(),
-                  labelText: '${context.l10n.maxEntities('Amount')}*',
+                  labelText: '${context.l10n.maxEntities(context.l10n.amount)}*',
```

Comment:
I still don't like translations wrapped into translations. Why not just making one string per label?

Reply:
I hope that i have fixed the issue now. I thought that it was the way to do it, but clearly I was wrong.